packages/gitmoji-changelog-core/src/index.js

```diff
@@ -28,7 +28,7 @@ function getCommits(from, to) {
 }
 
 function makeGroups(commits) {
```

Comment:
The sort problem you have come from here, you have to change `emoji` by `emojiCode` in the sort function:
https://github.com/frinyvonnick/gitmoji-changelog/blob/a0bc908f488ec896205ba96efcd7cba08d37bf67/packages/gitmoji-changelog-core/src/index.js#L37-L42

Reply:
Ha yeah, you're right.
Thank you!